ExtractComparisonCode.txt

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
```

Comment:
this file is not needed we can tell what the commit by seeing the commit message